packages/flutter/lib/src/rendering/object.dart

```diff
@@ -3317,12 +3317,16 @@ abstract class RenderObject with DiagnosticableTreeMixin implements HitTestTarge
   }
 
   /// {@template flutter.rendering.RenderObject.getTransformTo}
-  /// Applies the paint transform up the tree to `ancestor`.
+  /// Applies the paint transform up the tree to `target`.
```

Comment:
Is "up the tree" still correct here? It sounds like target can be anywhere in the tree, it doesn't necessarily have to by above `this`?